release-handbook/release_handbook.md

```diff
@@ -46,138 +48,179 @@ This information guides you to archive the documentation of the previous version
 
 The vNext release documentation is hosted in the `docs-staging` branch. The `docs-staging` branch is merged to the `master` branch when a release is published. 
 
-Once a new release rolls out, the `docs-staging` branch should be updated to get prepared for the next release. The updates include but are not limited to the following aspects:
+Once the Code Freeze for a new release takes place, the `docs-staging` branch must be updated to get prepared for the next release. The updates include but are not limited to the following aspects:
 
-- Archive the previous cversion documentation
+- Archive the previous version documentation
 - Bump the release version number to vNext
 - Create placeholder files and folders for the vNext release, such as release notes document and TPSR document
 
-The following steps takes v1.25 release prepration as an example. v1.24 was released a day ago, now we will prepare the `docs-staging` branch for the v1.25 release. An example commit for the steps can be found in [this](https://github.com/zowe/docs-site/commit/7f3dc7a938609d84f15d3ad6e67198155d0de029) commit. 
+The following steps takes v1.25 release preparation as an example. You should follow these steps the day Code Freeze takes place.
+### **Part 1: Sync the doc branches**
 
-### Part 1: Archive the previous release doc
+Summary: Update the `docs-staging` branch so it includes all the necessary updates made to the `master` branch.
 
-1. Update the `docs-staging` branch so it includes all the necessary updates made to the `master` branch. You may need to cherry pick changes to ensure the correct content is copied over to `docs-staging`.
-    - You can use GitHub Desktop to do this. Select the `Branch` menu, then the `Compare to branch` option. Select the `master` branch for comparison.
-2. Go to the `docs-staging` branch.
-3. Navigate to the `/docs-site` directory.
-4. Run the following command:
+**Procedure**
+1. Create and publish a new (temporary) branch based off the `master` branch.
+
+2. Create a PR to merge the temporary branch into the `docs-staging` branch.
+  
+    This syncs both branches so that `docs-staging` is updated with all the edits made to the `master` branch since the last time the branches were synched.
+
+3. Create a local build to confirm the updated `docs-staging` branch has no errors.
+
+    1. Run `npm install`.
+    2. Run `npm start` to build the site locally and clear any errors.
+
+4. Merge the pull request so that `docs-staging` matches the `master` branch.
```

Comment:
For this pull request (and any other PR that merges content from docs-staging into master or vice versa), we should add the note about _not_ squash merging:
> The engineer should **NOT** commit with a "squash and merge," which can lead to merge conflicts for later releases.

Reply:
done.